### PR TITLE
German Translation

### PR DESCRIPTION
**Pull request by BCHLOVER**
German Translation for bitcoincash.org

First time using GitHub do I don't know if I did this pull stuff correctly

### CODE REVIEW

**Comment by tbrannt** [comment]
Haha saw it too late that you already did it. Was also 3/4 through.. fast germans

**Comment by BCHLOVER** [comment]
Unlucky haha. If you have something that you find sounds better than mine be free to criticize, I'm in no way a professional writer :)

File: translations/de.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+        - |
+          Jeder, der bei Block 478558 Bitcoin besessen hat, erhielt ebenso automatisch Bitcoin Cash. Jeder ist willkommen, dabei zu helfen Bitcoin Cash für die gesamte Welt zugänglich zu machen.
+      features:
+        - |
+          <strong>On-Chain-Skalierung</strong> - Bitcoin Cash folgt dem Leitplan von Satoshi Nakamoto für globale Akzeptanz durch On-Chan-Skalierung. Als erster Schritt wurde das Blocksizelimit anpassbar festgelegt. Ebenso wurde das Standardlimit auf 8MB erhöht. <a href="https://github.com/BitcoinUnlimited/BUIP/blob/master/065.mediawiki" target="_blank">Forschung wird betrieben, um zukünftige Blocksize-Erhöhungen zu ermöglichen.</a>.
+        - |
+          <strong>New Transaktionssignaturen</strong> - Ein neuer SigHash-Typ bietet Replay Protection, verbesserte Walletsicherheit und Eliminierung des Quadratic-Hashing-Problems.
```
Comment (tbrannt) [inline translations/de.yml]: "Neue Transaktionssignaturen"

File: translations/de.yml [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+        - question: "Hat das Bitcoin Blocksizelimit von 1MB Probleme verursacht?"
+          answer:
+            - |
+              Ja, in 2017, als die Kapazität komplett ausgelastet wurde. Transaktionskosten sind explodiert und Bitcoin wurde unzuverlässig, viele Transaktionen wurden selbst nach mehreren Tagen nicht bestätigt.
+            - |
+              Bitcoin's Marktpreis ist gestiegen, aber das Wachstum und der Nutzen als Währung ist stagniert. Viele Nutzer, Händler, Geschäfte und Investoren haben Bitcoin für Alternativen verlassen, was dazu geführt hat, dass die Dominanz von Bitcoin von 95% auf bis zu 40% gefallen ist.
+        - question: "Behebt Bitcoin Cash diese Probleme?"
+          answer:
+            - |
+              Ja.Bitcoin Cash hat das Blocksizelimit als Teil der On-Chain Skalierungsherangehensweise auf 8MB erhöht, was genug Platz für jede Transaktion bietet.
```
Comment (tbrannt) [inline translations/de.yml]: Space after dot

**Comment by tbrannt** [comment]
I like your translations. You stuck closer to the english version, I translated some parts more freely, e.g.: 

description: "Bitcoin Cash eröffnet der Welt die Nutzung einer neuen Art von Geld. Händler sowie Nutzer sind durch dieses in der Lage schnelle, günstige und zuverlässige Transaktionen zu tätigen. Vor uns liegt eine vielversprechende Zukunft - ohne künstlich eingeschränktes Wachstum - mit zugangsfreier Innovation und dezentralisierter Weiterenwicklung von Bitcoin Cash."

File: translations/de.yml [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+        - |
+          <strong>New Transaktionssignaturen</strong> - Ein neuer SigHash-Typ bietet Replay Protection, verbesserte Walletsicherheit und Eliminierung des Quadratic-Hashing-Problems.
+        - |
+          <strong>Neuer Difficulty Adjustment Algorithmus</strong> - Der neue Algorithmus erlaubt eine schnelle Anpassung der Difficulty, schützt vor Hashrateschwankungen und erlaubt Minern fließend von Legacy Bitcoin Blockchain auf Bitcoin Cash umzusteigen.
+        - |
+          <strong>Dezentralisierte Entwicklung</strong> -  Mit mehreren unabhängigen Entwicklungsteams, die Softwareimplementierungen bieten, ist die Zukunft gesichert. Bitcoin Cash ist resistent gegen politische oder soziale Attacken auf die ENtwicklung des Protokolls. Keine einzelne Gruppe oder einzelnes Projekt kann es kontrollieren. Die<a href="https://lists.linuxfoundation.org/mailman/listinfo/bitcoin-ml" target="_blank">bitcoin-ml Mailingliste</a> ist eine gute Plattform um Vorschläge für Änderungen anzubringen, die Koordination und Zusammenarbeit von mehreren Entwicklungsteams benötigt.
```
Comment (tbrannt) [inline translations/de.yml]: würde hier auch "Entwicklerteams" anstatt "Entwicklungsteams" schreiben

File: translations/de.yml [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        - |
+          Jeder, der bei Block 478558 Bitcoin besessen hat, erhielt ebenso automatisch Bitcoin Cash. Jeder ist willkommen, dabei zu helfen Bitcoin Cash für die gesamte Welt zugänglich zu machen.
+      features:
+        - |
+          <strong>On-Chain-Skalierung</strong> - Bitcoin Cash folgt dem Leitplan von Satoshi Nakamoto für globale Akzeptanz durch On-Chan-Skalierung. Als erster Schritt wurde das Blocksizelimit anpassbar festgelegt. Ebenso wurde das Standardlimit auf 8MB erhöht. <a href="https://github.com/BitcoinUnlimited/BUIP/blob/master/065.mediawiki" target="_blank">Forschung wird betrieben, um zukünftige Blocksize-Erhöhungen zu ermöglichen.</a>.
+        - |
+          <strong>New Transaktionssignaturen</strong> - Ein neuer SigHash-Typ bietet Replay Protection, verbesserte Walletsicherheit und Eliminierung des Quadratic-Hashing-Problems.
+        - |
+          <strong>Neuer Difficulty Adjustment Algorithmus</strong> - Der neue Algorithmus erlaubt eine schnelle Anpassung der Difficulty, schützt vor Hashrateschwankungen und erlaubt Minern fließend von Legacy Bitcoin Blockchain auf Bitcoin Cash umzusteigen.
```
Comment (tbrannt) [inline translations/de.yml]: "fließend von **der** Legacy Bitcoin Blockchain"

File: translations/de.yml [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+            - |
+              Bitcoin Cash ist eine elektronische Peer-to-Peer Währung für das Internet. Es ist komplett dezentralisiert, ohne Zentralbank und benötigt kein Vertrauen in Drittpartein um zu funktionieren.
+        - question: "Ist Bitcoin Cash anders als "Bitcoin"?"
+          answer:
+            - |
+              Ja. Bitcoin Cash ist die Weiterführung von Bitcoin als elektronische Peer-to-Peer Währung. Es ist eine Fork der Bitcoin Blockchain mit aktualisierten Regeln, die es Bitcoin Cash erlauben zu wachsen.
+        - question: "Besitze ich auch Bitcoin Cash, falls ich bereits Bitcoin besitze?"
+          answer:
+            - |
+              Jeder, der Bitcoin zum Zeitpunkt der Erstellung von Bitcoin Cash besessen hat, wurde automatisch zu einem Bitcoin Cash Besitzer. Das heißt, dass bei Block 478558 (1. August, ca. 13:16 UTC) Bitcoinbesitzer die gleiche Menge wie Bitcoin an Bitcoin Cash erhalten haben. Falls Ihre Bitcoins zu diesem Zeitpunkt bei einer Drittpartei, wie z.B. einer Börse war müssen Sie sich bei dieser nach ihren Bitcoin Cash erkundigen.
```
Comment (tbrannt) [inline translations/de.yml]: "[...] wie z.B. einer Börse war**en,** müssen Sie sich bei dieser [...]"

File: translations/de.yml [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+            - |
+              Jede Transaktion nach dem 1. August wird von Bitcoin und Bitcoin Cash seperat verarbeitet. Dies bedeutet, dass Bitcoin, die danach erworben wurden, keine Bitcoin Cash enthalten und Bitcoin Cash, die dnaach erworben wurden keine Bitcoin enthalten.
```
Comment (tbrannt) [inline translations/de.yml]: "dnaach" -> "danach"

**Comment by BCHLOVER** [comment]
Thank you very much for the fixes. I wrote it in Notepad++, which didn't highlight my fatfinger errors. As for the close to the source translation, do you think it sounds forced, or is it natural? I wasn't sure how close to the source it should be.

**Comment by tbrannt** [comment]
I'm sure my translations didn't contain less errors. I think it's fine to be a bit closer to the english version as I was.

**Comment by zquestz** [comment]
A german translation was provided in the main repo at https://github.com/bitcoincashorg/website. This now has conflicts but I would love if you would go over the existing translation and provide updates/comments. Thank you for your help!

**Comment by BCHLOVER** [comment]
Sorry, but that translation is horrible. This guy also seems to have skipped every german lesson in school because he uses no commas. Enjoy a subpar translation, that was added later than mine.

**Comment by zquestz** [comment]
Since you both worked on this together I am happy to use the updated version. Seems more work was done on it. Could you just repost on the actual site? Technically I shouldn't be merging PR's here, this is just my work area. 

I just didn't see this one yet because I always check the main repo for submissions first. Didn't mean to be rude.

**Comment by BCHLOVER** [comment]
I hope I added it correctly this time I'm new to github. Sorry if I was rude, I was just a bit dumbfounded because this translation was "reviewed" by tbrannt and had things changed.

**Pull request closed without merging**